docs/mms/index.md

```diff
@@ -0,0 +1,91 @@
+# Model Server for Apache MXNet (incubating)
+
+Model Server for Apache MXNet (incubating), otherwise known as MXNet Model Server (MMS), is an open source project aimed at providing a simple yet scalable solution for model inference. It is a set of command line tools for packaging model archives and serving them. The tools are written in Python, and have been extended to support containers for easy deployment and scaling. MMS also supports basic logging and advanced logging with AWS CloudWatch integration.
```

Comment:
Should we say MMS supports both Gluon and MXNet symbolic models along with ONNX models?